packages/server/src/routes/predictions/index.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express'
+import predictionsController from '../../controllers/predictions'
+const router = express.Router()
+
+// CREATE
+router.post(
+    '/:id',
+    predictionsController.uploadFilesMiddleware,
```

Comment:
we know this upload middleware is not working right? we'll need to change to same logic as `routes/vectors/index.ts`

Reply:
and also remove this middleware from `controllers`